src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -33,6 +33,8 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+import static htsjdk.variant.variantcontext.VariantContextUtils.calculateChromosomeCounts;
```

Comment:
I find static imports a bit hard to follow in the code, not sure what the GATK style is though (but I haven't seen it very often)